libs/ledger-live-common/src/account/serialization.ts

```diff
@@ -78,9 +78,9 @@ export const fromOperationRaw = (
     const family = inferFamilyFromAccountId(operationRaw.accountId);
 
     if (family) {
-      const specific = familySpecific[family];
-      if (specific && specific.fromOperationExtraRaw) {
-        operation.extra = specific.fromOperationExtraRaw(operationRaw.extra);
+      const bridge = getAccountBridgeByFamily(family, accountId);
```

Comment:
Maybe simpler to define `getAccountBridge(accountId: string)` and call `inferFamilyFromAccountId` inside of it?

Reply:
I don't see how you can retrieve an `AccountBridge` with only the `accountId`. AFAIK you need the `family` to retrieve an `AccountBridge`

Reply:
I was just thinking of changing `getAccountBridgeByFamily` to basically:
```
export function getAccountBridgeFromId(accountId: string): AccountBridge<any> {
  const family = inferFamilyFromAccountId(accountId);

  if (accountId) {
    const { type } = decodeAccountId(accountId);

    if (type === "mock") {
      const mockBridge = mockBridges[family];
      if (mockBridge) return mockBridge.accountBridge;
    }
  }

  const jsBridge = jsBridges[family];
  if (!jsBridge) {
    throw new CurrencyNotSupported("currency bridge not found " + family);
  }
  return jsBridge.accountBridge;
}
```

But not very important, you can skip this one if you want.